exercises/protein-translation/protein_translation_test.go

```diff
@@ -11,6 +12,9 @@ type codonCase struct {
 	errorExpected error
 }
 
+var MockErrStop = errors.New("stop")
+var MockErrInvalidBase = errors.New("invalid base")
```

Comment:
I think the correct approach would be create a custom type like this:
```go
type MockErrInvalidBase struct{}

func (m *MockErrInvalidBase) Error() string {
    return "invalid base"
}


type MockErrStop struct{}

func (m *MockErrStop) Error() string {
    return "stop"
}

```
And then you compare the types to assert the correct type is returned
More info: https://www.digitalocean.com/community/tutorials/creating-custom-errors-in-go

Reply:
Yes, that piece is helpful, but a little out of date: since Go 1.13, we should use `errors.As` (for checking against custom error types) and `errors.Is` (for checking against sentinel errors, but we shouldn't use those: we should use custom error types instead).

Reply:
Could you mark all resolved comments as resolved, @GLOFonseca?